dataactvalidator/config/sqlrules/a36_appropriations.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
+    approp.row_number,
+    approp.budget_authority_unobligat_fyb,
+    sf.amount as sf_133_amount
```

Comment:
Similar question here as for the A34 rule..should we make the column alias more explicit?